docs/source/conf.py

```diff
@@ -210,7 +210,7 @@
 .. _NCBI BLAST Cloud Documentation: http://ncbi.github.io/blast-cloud/
 .. _R: http://www.r-project.org/
 .. _S3IT: http://www.s3it.uzh.ch/
-.. _SLURM: https://computing.llnl.gov/linux/slurm/
+.. _Slurm: https://slurm.schedmd.com/
```

Comment:
The grid engine link ( .. _Grid Engine: http://gridengine.info/) needs an update.